prow/config.yaml

```diff
@@ -14626,35 +14298,23 @@ periodics:
   name: cluster-registry-nightly
   labels:
     preset-service-account: true
+    preset-bazel-scratch-dir: true
   spec:
     containers:
-    - image: gcr.io/k8s-testimages/bazelbuild:v20180201-0184a54dc-0.8.1
+    - image: gcr.io/k8s-testimages/kubekins-e2e:v20180307-ce8e19913-1.10
```

Comment:
seems unrelated but i guess we are just making sure all images are the latest. right?

Reply:
yeah, I should have left a note about this, the cluster registry presubmits use this now, but CI slipped through.